android/src/app/organicmaps/bookmarks/BookmarkCategoriesFragment.java

```diff
@@ -208,7 +209,9 @@ public void onBookmarksLoadingFinished()
   @Override
   public void onBookmarksFileLoaded(boolean success)
   {
-    // Do nothing here.
+    View view = getActivity().findViewById(R.id.fragment_container);
+    if (view != null)
+      Utils.showSnackbar(requireActivity(), view, null, success ? R.string.load_kmz_successful : R.string.load_kmz_failed);
```

Comment:
There is the same onBookmarksFileLoaded callback in the MwmActivity.java. Why is it not called in these cases?

Reply:
Looks like mwmactivity has code 
```
BookmarkManager.INSTANCE.removeLoadingListener(this);
```
in Onstop function and that line is changed by [this](https://github.com/lapaz17/organicmaps/commit/bee0b712747b4882c2d50a72ff6df7794863f5fe) commit. Since mwmactivity is superseded with bookmarkcategoriesactivity, onStop method of it being called. I honestly dont know what to do next.

Reply:
Does that mean that we should support both listeners? Or we can remove one in MWMActivity?

Reply:
We can remove one in MWM. I will & commit it.